src/service-override/views.ts

```diff
@@ -189,6 +191,10 @@ function renderStatusBarPart (container: HTMLElement): IDisposable {
   return attachPart(Parts.STATUSBAR_PART, container)
 }
 
+interface BodyRenderer extends IDisposable {
+  setInput? (input: EditorInput, options: IEditorOptions | undefined, context: IEditorOpenContext, token: CancellationToken): Promise<void>
```

Comment:
What about providing a service accessor as parameter? otherwise there is no simple way to get the FileService to get the file content